apps/chef_db/src/chef_sql.erl

```diff
@@ -109,6 +109,7 @@
 
 -include_lib("sqerl/include/sqerl.hrl").
 -include("../../include/chef_types.hrl").
+-include("../../include/oc_chef_types.hrl").
```

Comment:
Is this needed? I don't see anything in this change that references the things defined in oc_chef_types.hrl.

(As a side note, with all these projects merged, there's little if any benefit to having oc_chef_types distinct from chef_types - probably worth combining those into chef_types.hrl.  I'll make a separate PR for that... )